packages/cli/package.json

```diff
@@ -85,16 +85,16 @@
   "config": {
     "templateDependencies": {
       "@types/mocha": "^5.2.7",
-      "@types/node": "^10.17.3",
+      "@types/node": "^10.17.5",
       "cross-spawn": "^7.0.1",
       "debug": "^4.1.1",
       "fs-extra": "^8.1.0",
       "mocha": "^6.2.2",
       "nyc": "^14.1.1",
-      "prettier": "^1.18.2",
+      "prettier": "^1.19.1",
       "rimraf": "^3.0.0",
       "source-map-support": "^0.5.16",
-      "typescript": "~3.6.4",
+      "typescript": "~3.7.2",
```

Comment:
I am surprised to see this change as part of the commit `chore: refresh package-lock.json`, I would expect it to be a part of `chore: upgrade to TypeScript 3.7.2`.